### DISCUSSION TITLE
New LLM Support: groq

### DISCUSSION THREAD

**Discussion by loveunk** [immersive-translate/immersive-translate#1509] [Ideas]
### 新功能 | New Feature

[Groq](https://groq.com/)'s inference speed is so fast, hope it get supported soon. Thank you.

### 为什么您想要添加这些功能 | Why you want to add these features

[Groq](https://groq.com/)'s inference speed is so fast, hope it get supported soon. Thank you.

**Comment by keyboard3** [comment]
You can refer to [this document](https://immersivetranslate.com/docs/services/ai/) for temporary access
